Example/CotterIOS/EntryViewController.swift

```diff
@@ -121,6 +128,23 @@ class EntryViewController: UIViewController {
         })
     }
     
+    @IBAction func enrollTrustedDeviceForUser(_ sender: Any) {
```

Comment:
Why do you need this here?

Reply:
cuz my phone was already a registered user but i haven't enrolled in trusted devices yet. Can prob take it out if u dont like